### PR TITLE
Add Alfen Evcs

### PR DESCRIPTION
**Pull request by TimStoecker**
*Add Alfen Evcs Based on PR#28

### CODE REVIEW

**Comment by yvesilknievil** [comment]
Hi @TimStoecker 

Thanks for acting so quick on this topic.
I merged it to my fork, and do notice some strange behaviour:
When I start charging, it starts and then after a while the controller says: battery full and the charging stops.
The cycle then repeats itself.

Anything I would've missed?

**Comment by TimStoecker** [comment]
Hi @yvesilknievil 

Thank you for testing of the Alfen Implementation. I checked again and the translation of the Status is not well explained in the documentation. It is possible that the previous mapping of the register was wrong. I changed it to the one of the IEC62196 specification.
Let me know if the error persists!

File: io.openems.edge.evcs.alfen/src/io/openems/edge/evcs/alfen/EvcsAlfen.java [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+		CHARGE_POWER_L1(Doc.of(OpenemsType.FLOAT).unit(Unit.WATT) //
+				.persistencePriority(PersistencePriority.HIGH) //
+				.text("Charge Power L1")), //
+		CHARGE_POWER_L2(Doc.of(OpenemsType.FLOAT).unit(Unit.WATT) //
+				.persistencePriority(PersistencePriority.HIGH) //
+				.text("Charge Power L2")), //
+		CHARGE_POWER_L3(Doc.of(OpenemsType.FLOAT).unit(Unit.WATT) //
+				.persistencePriority(PersistencePriority.HIGH) //
+				.text("Charge Power L3")), //
+		APPARENT_POWER_L1(Doc.of(OpenemsType.FLOAT).unit(Unit.VOLT_AMPERE) //
```
Comment (clehne) [inline io.openems.edge.evcs.alfen/src/io/openems/edge/evcs/alfen/EvcsAlfen.java]: We should remove apparentPowerL1,2,3, reactivePowerL1,2,3, energy deliveredL1,2,3/consumedL1,2,3. They are unnessesary for all usecases I can think off.

**Review by clehne — CHANGES_REQUESTED** [review]

**Comment by clehne** [comment]
Like that bundle. Please remove the unnessesary channels and rebase (due to merged #28).

**Review by clehne — APPROVED** [review]
Great.

**Pull request merged**